vendor/src/github.com/vbatts/tar-split/archive/tar/common.go

```diff
@@ -139,8 +139,8 @@ func (fi headerFileInfo) Mode() (mode os.FileMode) {
 	}
 
 	switch fi.h.Typeflag {
-	case TypeLink, TypeSymlink:
```

Comment:
this is from go1.5 updates to `archive/tar` https://github.com/vbatts/tar-split/commit/b4d27b54261d4a30c6f6a0967cba8b4a742d19bc